src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class StorageDaoImpl implements StorageDao {

```

Comment:
DAO - Data Access Object Pattern (feel free to google). 

should be responsible only for interaction with database (the storage in your code). 
Any other code should be present somewhere else (Service Layer for example). 

Now this class violates SRP (Single Responsibility Principle of SOLID).